lib/utils/generateFinalBridgeObject.ts

```diff
@@ -0,0 +1,42 @@
+import type { ISbStoriesParams, StoryblokBridgeConfigV2 } from "@storyblok/js";
+
+export interface RawCodeItem {
+  options?: {
+    apiOptions?: ISbStoriesParams;
+    bridgeOptions?: StoryblokBridgeConfigV2;
+  };
+}
+export function generateFinalBridgeObject(
+  rawCode: RawCodeItem[]
+): StoryblokBridgeConfigV2 {
```

Comment:
I'd move the return type to the actual returned object, instead of the function return. Something like
```
  let mergedOptions: StoryblokBridgeConfigV2 = {
    resolveRelations: [],
  };
```
That's because currently the `mergedOptions` variable is typed as `any` and this could trick us into believing that Typescript is doing the heavy lifting while it's not. WDYT?